build.properties

```diff
@@ -2,10 +2,8 @@
 # Don't directly modify this file. Instead, copy it to local.build.properties and
 # edit that.
 #
-# $Id$
 project.name = eXist-db
-project.version = 3.0
```

Comment:
@shabanovd Yes, thanks, like I wrote this is used for all package related  sem-ver checks both in package manager (dashboard) and public repo apps.